.yarn/sdks/typescript/lib/tsserver.js

```diff
@@ -109,15 +109,13 @@ const moduleWrapper = tsserver => {
         return process.platform === `win32`
           ? str.replace(/^.*zipfile:\//, ``)
           : str.replace(/^.*zipfile:/, ``);
-      } break;
-
+      }
```

Comment:
this isn't part of check-spelling, but it is something I would encourage in my other lives... the `break` is unreachable because the `return` ends the code path.